File: Python PowerPoint Examples/TableApiDemo/setSlideTableCellTextFormat.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudpowerpoint.api.table_api.TableApi(configuration)

fileName = "SetSlideTableCellTextFormat.pptx"
password = ""
storage = ""
slide_index = 0
shape_index = "0"
path = "[0,0]"
source = spirecloudpowerpoint.models.Portion()
source.text = "11111111111111"
source.font_color = "lightpink"
source.font_italic = True
source.spacing = 2
inputFolder = "input"
result =api.set_slide_table_cell_text_format(fileName, slide_index, shape_index, path,source=source,password=password, storage=storage, folder=inputFolder)

File: JavaScript PowerPoint Examples/TableApiDemo/setSlideTableCellTextFormat.js
(function (){
    var appId = "your ID";
    var appKey = "your Key";
    var baseUrl = "https://api.e-iceblue.cn";

    var SpirecloudPowerpoint = require('../../src/index');
    var apiClient = new SpirecloudPowerpoint.ApiClient(appId, appKey,baseUrl);
    var instance = new SpirecloudPowerpoint.TableApi(apiClient);
    
    var name = "SetSlideTableCellTextFormat.pptx";
    var folder = "input";
    var slideIndex = 0;
    var shapeIndex = 0;
    var path = "[0,0]";
    var opts = {};
    opts.source = new SpirecloudPowerpoint.Portion();
    opts.source.Text = "11111111111111";
    opts.source.FontItalic = true;
    opts.source.Spacing = 2.0;
    opts.source.FontColor = "lightpink";

    opts.folder = folder;
    var response = instance.setSlideTableCellTextFormat(name, slideIndex, shapeIndex, path, opts,
        function (error, data, response) {
            if (error) throw error;
        });
})();